cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,30 @@
-describe('', () => {
-  before(() => {
-
+describe('The Article page', () => {
+  let user;
+  let article;
+  beforeEach(() => {
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+    cy.task('generateArticle').then(generateArticle => {
+      article = generateArticle;
+    });
   });
 
-  it('', () => {
+  it('should allow creating an article', () => {
+    cy.login(user);
+  
+    cy.createArticle(article);
 
+    cy.checkArticle(article, user);
   });
 
-  it('', () => {
+  it('should allow deleting an article', () => {
+    cy.login(user);
+  
+    cy.createArticle(article);
```

Comment:
```suggestion
    cy.createArticle(article);
```
try to extract slug from this response and visit the article page directly